compose/mpp/demo/src/commonMain/kotlin/androidx/compose/mpp/demo/bug/BugReproducers.kt

```diff
@@ -27,5 +27,6 @@ val BugReproducers = Screen.Selection(
     IOSDynamicKeyboardType,
     NoPressInteractionInOutlinedTextField,
     WebBaselineAlways0,
-    ResizePopupCrashOnJS
+    ResizePopupCrashOnJS,
+    KeyEventMappingsBugReproducer
```

Comment:
We agreed that it's better to replace mpp "bug reproducers" to auto tests step by step. If you're adding a test for the case, no need to add it to mpp

Reply:
fixed